devtools/create_certs.py

```diff
@@ -27,16 +27,30 @@ def create_key_and_csr(key, csr):
     ])
 
 
-def create_crt(csr, crt, root_ca_crt, root_ca_key):
-    run(['openssl', 'x509', '-req',
+def create_crt(csr, crt, root_ca_crt, root_ca_key, entity, out_dir):
+    openssl_cmd = ['openssl', 'x509', '-req',
          '-in', csr,
          '-CA', root_ca_crt,
          '-CAkey', root_ca_key,
          '-CAcreateserial',
          '-out', crt,
          '-sha256',
-         '-days', '365',
-         '-extfile', 'ssl.ext'])
+         '-days', '365']
+    if (entity is 'node'):
```

Comment:
@mxm no I think this is necessary since the server wants to know to which domain it belongs to.